core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -1790,6 +1790,13 @@ private long loadMetadataAndJournal(LockedInodePath inodePath, LoadMetadataOptio
     UnderFileSystem ufs = resolution.getUfs();
     try {
       if (!ufs.exists(ufsUri.toString())) {
+        // The root is special as it is considered as PERSISTED by default.
```

Comment:
I mentioned before. I accidentally branched from the other PR. That should be checked in first. And then I will rebase this.